src/adhocracy_mercator/adhocracy_mercator/scripts/export_users.py

```diff
@@ -43,27 +88,39 @@ def export_users():
 
     users = query.execute()
 
+    # FIXME: set 100 instead of 0 on prod
```

Comment:
style guide:

- one abstrcation level per function
- short  (5 lines if possible), split in helper functions with lower abstraction level instead
- not more than 2 indention levels
- seperate error handling / command line options handling from actually doing something (not usefull for very simple functions)